From 553df35a405fccd679bdb16e65c79deb84928865 Mon Sep 17 00:00:00 2001
From: Alexandre Rostovtsev <tetromino@gmail.com>
Date: Wed, 7 Sep 2011 03:21:42 -0400
Subject: [PATCH] Disable tests that cannot be run from portage

Disable tests with modal dialogs and the tasks test that requires for
packagekit to preferentially use the dummy backend.
---
 src/gpk-self-test.c |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/gpk-self-test.c b/src/gpk-self-test.c
index f12ce73..ba01649 100644
--- a/src/gpk-self-test.c
+++ b/src/gpk-self-test.c
@@ -53,10 +53,10 @@ main (int argc, char **argv)
 	gpk_common_test (test);
 //	gpk_dbus_test (test);
 	gpk_language_test (test);
-	gpk_error_test (test);
+//	gpk_error_test (test);
 //	gpk_client_test (test);
-	gpk_modal_dialog_test (test);
-	gpk_task_test (test);
+//	gpk_modal_dialog_test (test);
+//	gpk_task_test (test);
 
 	return egg_test_finish (test);
 }
-- 
1.7.6.1

